sotodlib/tod_ops/sub_polyf.py

```diff
@@ -0,0 +1,190 @@
+import numpy as np 
+import matplotlib.pyplot as plt 
+
+from sotodlib import flags
+
+
+
+def get_turnaround_flags_by_dazdt(
+    tod, qlim=5.,az=None
```

Comment:
I'm not a fan of making copies of these functions that do the same math with different outputs. Especially not with flags / ranges objects. `ranges.mask()` will get you the boolean array for the flag and that code is written in C++ so it should be very fast. I don't see a reason to have these copies.

Reply:
Thanks for comments and a helpful information. 
I have not known that we have function `ranges.mask()`.
Now I deleted copies,  and that function is used in the current version.